nixos/modules/config/users-groups.nix

```
@@ -273,6 +273,9 @@ let
           {command}`passwd` command. Otherwise, it's
           equivalent to setting the {option}`hashedPassword` option.
 
+          Note that the {option}`hashedPassword` option will still
+          override this option if both are set.
```

I'm unsure where the "still" comes into play here, which reads as if it's mentioned before this line.

```suggestion
          Note that the {option}`hashedPassword` option will
          override this option if both are set.
```

---

Fixed, thanks!